src/test/typescript/io-spec.ts

```diff
@@ -0,0 +1,26 @@
+import { IO } from 'monet';
+
+const logX = IO(() => console.log('X'));
+const getY = logX.map(() => 'Y');
+const getOne = getY.flatMap(y => IO.of(() => y === 'Y' ? 1 : 0));
```

Comment:
It's just to test if typings work well. It's not an example of real life code :)